superset-frontend/src/components/Table/Table.overview.mdx

```diff
@@ -183,14 +183,27 @@ The table displays a set number of rows at a time, the user navigates the table
 The default page size and page size options for the menu are configurable via the `pageSizeOptions` and `defaultPageSize` props.
 NOTE: Pagination controls will only display when the data for the table has more records than the default page size.
 
-<Story id="design-system-components-table-examples--many-columns" />
+<Story id="design-system-components-table-examples--pagination" />
 
 ```
 <Table pageSizeOptions={[5, 10, 15, 20, 25] defaultPageSize={10} />
 ```
 
 ---
 
+### Virtualization for Performance
+
+The Table virtualization can be used to enable viewing data with many columns and or rows without paging.
```

Comment:
I think we can disassociate virtualization and pagination. You can have a paginated table with column virtualization. 
```suggestion
Table virtualization can enable viewing data with many columns and/or rows.
```

Reply:
makes sense